web/embed_style.html

```diff
@@ -268,4 +276,14 @@
     color: #eee;
     padding-left: 5px;
   }
+  
+  #default-doc-text {
```

Comment:
Can you make this a class selector instead of an id one, and apply the same class to both the doc title and console title div?